src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1699,7 +1699,11 @@ public async Task Http2_PingKeepAlive(TimeSpan keepAlivePingDelay, HttpKeepAlive
                     }
 
                     // Let connection live until server finishes.
-                    await Task.WhenAny(serverFinished.Task, Task.Delay(pingTimeout * 3));
+                    try
+                    {
+                        await serverFinished.Task.WaitAsync(pingTimeout * 3);
+                    }
+                    catch (TimeoutException) { }
```

Comment:
this is not the first place where we are ignoring the `TimeoutException`. Should we introduce a new test helper method for that? Sth like `TryWaitAsync`?

Reply:
There's only two places it would be relevant, and in two different test projects.  I don't think it's worth a shared helper.